include/envoy/ssl/context_config.h

```diff
@@ -88,6 +88,11 @@ class ClientContextConfig : public virtual ContextConfig {
    * @return The maximum number of session keys to store.
    */
   virtual size_t maxSessionKeys() const PURE;
+
+  /**
+   * @return const std::string& with the signature algorithms for the context.
```

Comment:
There is no standard string representation of a signature algorithm list. This should document what format is being used.